build/azure-pipelines/common/extract-telemetry.sh

```diff
@@ -2,29 +2,19 @@
 set -e
 
 cd $BUILD_STAGINGDIRECTORY
-git clone https://github.com/microsoft/vscode-telemetry-extractor.git
-cd vscode-telemetry-extractor
-git checkout 4e64f3de30f8fccb58ebdc0d85c4861a135d46cf
-npm i
-npm run compile
-cd src
-mkdir telemetry-sources
-cd telemetry-sources
+mkdir extraction
+cd extraction
+npm install vscode-telemetry-extractor@1.3.1
```

Comment:
Shouldn't `vscode-telemetry-extractor` just be a dev dependency listed in `build/package.json`?

Reply:
I didn't want people running vs code from sources on their local machine to have to download the module and all its dependencies since it really is just a tool for the build machine and doesn't make sense in a local development scheme. So I just place it in its own folder run it and then cleanup all traces of it so that the node modules don't get scanned and mess up component governance

Reply:
That is fine. We already do that with many other modules. I really would like this dependency (and all its dependencies) be listed in a `package.json` file (and `yarn.lock` file), to track everything we're using. We do want to use component governance there.

If you still want to avoid that a dev would have to spend time downloading it, I suggest creating a `build/azure-pipelines/telemetry` folder and placing that `package.json` and `yarn.lock` in there. The build machine would then `cd` to that and `yarn`.

Reply:
Alright I moved it over.